### ISSUE TITLE
macOS Installer has 200 MB – Why is it so big?

### ISSUE THREAD

**Issue by jirir** [mullvad/mullvadvpn-app#3085]
## Issue report

Operating system: macOS 11.6.1

App version: 2021.5

### Issue description

The .pkg is giantic 200 MB. Why is it so big?

**Comment by faern** [comment]
It doubled in size once we made it a universal installer containing two versions of the app in one package: Both the Intel and the Apple Silicon versions of the app. Cut in half it's roughly the same size as the other platforms.

We decided to make one big installer rather than two separate ones, one for Intel macs and one for Apple Silicon macs, because people usually prefer to keep it simple. Most people have enough bandwidth and disk space to not care about 200 MiB files. But needing to figure out which one of two files to download would be harder.
